server/api/users.go

```diff
@@ -62,10 +62,26 @@ func (a *API) handleGetUsersList(w http.ResponseWriter, r *http.Request) {
 	auditRec := a.makeAuditRecord(r, "getUsersList", audit.Fail)
 	defer a.audit.LogRecord(audit.LevelAuth, auditRec)
 
-	users, err := a.app.GetUsersList(userIDs)
-	if err != nil {
-		a.errorResponse(w, r.URL.Path, http.StatusBadRequest, err.Error(), err)
-		return
+	var users []*model.User
+	var error error
+	if userIDs[0] == model.SingleUser {
```

Comment:
I think you need to check if userIDs is an empty list first, otherwise it will panic.